UpstreamTagger/UpstreamTagger.h

```diff
@@ -103,9 +102,7 @@ class UpstreamTagger: public FairDetector
     virtual void   BeginEvent() {;}
    
     Double_t module[11][3];
-    
-  private:
-
```

Comment:
Making the UpstreamTagger and its hits [friends](https://en.cppreference.com/w/cpp/language/friend) might be a better solution than making the members public.